packages/okta-auth-js/lib/pkce.js

```diff
@@ -53,6 +54,18 @@ function saveMeta(sdk, meta) {
 function loadMeta(sdk) {
   var storage = getStorage(sdk);
   var obj = storage.getStorage();
+  // Verify the Meta
+  if (!obj.codeVerifier) {
+    // If meta is not valid, try reading from localStorage.
+    // This is for compatibility with older versions of the signin widget.
```

Comment:
I've a Jira ticket reference here so someone could look up the specifics and know if the comment still applies.